source/main/resources/CacheSystem.cpp

```diff
@@ -1138,8 +1140,7 @@ void CacheSystem::LoadResource(CacheEntry& t)
         return;
     }
 
-    static int rg_counter = 0;
-    String group = "Mod-" + std::to_string(rg_counter++);
+    String group = t.fname;

```

Comment:
Well, there's non-zero chance this will cause conflicts, I can imagine thins like `woodenbox.load`